pkg/ebpf/bpf_errors.go

```diff
@@ -0,0 +1,228 @@
+package ebpf
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+	"strings"
+	"unsafe"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+const BPF_MAX_ERR_FUNC_LEN = 48
+
+type BPFErrorType uint32
+
+const (
+	BPF_ERR_MAP_UPDATE_ELEM BPFErrorType = iota + 1
+	BPF_ERR_MAP_DELETE_ELEM
+	BPF_ERR_GET_CURRENT_COMM
+	BPF_ERR_TAIL_CALL
+
+	BPF_ERR_UNSPEC BPFErrorType = math.MaxUint32
```

Comment:
nit: Since 0 is the deafult value used in many cases, you can set unspec to be 0 (iota) as @NDStrahilevitz suggested in one of his comments.